common/app/layout/Sublink.scala

```diff
@@ -211,8 +211,8 @@ object FaciaCard {
     showSeriesAndBlogKickers: Boolean
   ): FaciaCard = {
 
-    val isTestContainer = config.displayName.contains("Paid content in unbranded container")
-    if (isTestContainer && faciaContent.branding(defaultEdition).exists(_.isPaid)) {
+    val containerName: Option[String] = config.displayName
+    if ((containerName.contains("Paid content in unbranded container") || containerName.contains("lifestyle")) && faciaContent.branding(defaultEdition).exists(_.isPaid)) {
```

Comment:
Might be good to put a feature switch around this condition so that we either remove the container name condition or the entire functionality when the switch expires. (Depending on the outcome of the test). (Which will be success.)